javascript/javascript1/week1/homework/ageCalculator.js

```diff
@@ -0,0 +1,22 @@
+console.log("====Human Age Checker====")
+/* human age calculator */
+const yearOfBirth = 1994;
+const yearFuture = 2027;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be ", age, "years old in ", yearFuture);
```

Comment:
This logs as `"You will be " 33 "years old in " 2027` - not super pretty with the extra `"`. To improve on it, you could use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
```suggestion
console.log(`You will be ${age}, years old in ${yearFuture}`);
```